weaver/wps_restapi/swagger_definitions.py

```diff
@@ -466,26 +466,78 @@ class LandingPage(ExtendedMappingSchema):
     links = LinkList()
 
 
+class FormatMimeType(ExtendedMappingSchema):
+    """
+    Used to respect ``mimeType`` field to work with pre-existing processes
+
+    .. seealso::
+        - :class:`FormatSelection`
+        - :class:`DeployFormatDefaultMimeType`
+    """
+    title = "Format"
+    mimeType = ExtendedSchemaNode(
+        String(), default=CONTENT_TYPE_TEXT_PLAIN, example=CONTENT_TYPE_APP_JSON)
+    schema = ExtendedSchemaNode(String(), missing=drop)
+    encoding = ExtendedSchemaNode(String(), missing=drop)
+
+
 class Format(ExtendedMappingSchema):
+    """
+    Used to respect ``mediaType`` field as suggested per `OGC-API`
+
+    .. seealso::
+        - :class:`FormatSelection`
+        - :class:`DeployFormatDefault`
+    """
     title = "Format"
-    mimeType = ExtendedSchemaNode(String())
+    mediaType = ExtendedSchemaNode(
+        String(), default=CONTENT_TYPE_TEXT_PLAIN, example=CONTENT_TYPE_APP_JSON)
     schema = ExtendedSchemaNode(String(), missing=drop)
     encoding = ExtendedSchemaNode(String(), missing=drop)
 
 
-class FormatDefault(Format):
-    """Format for process input are assumed plain text if the MIME-type was omitted and is not
+class DeployFormatDefaultMimeType(FormatMimeType, ExtendedMappingSchema):
+    """
+    Format for process input are assumed plain text if the MIME-type was omitted and is not
+    one of the known formats by this instance. When executing a job, the best match will be used
+    to run the process, and will fallback to the default as last resort.
+    """
+    mimeType = ExtendedSchemaNode(
+        String(), example=CONTENT_TYPE_APP_JSON)
+
+
+class DeployFormatDefault(Format, ExtendedMappingSchema):
+    """
+    Format for process input are assumed plain text if the MEDIA-type was omitted and is not
     one of the known formats by this instance. When executing a job, the best match will be used
     to run the process, and will fallback to the default as last resort.
     """
-    mimeType = ExtendedSchemaNode(String(), default=CONTENT_TYPE_TEXT_PLAIN, example=CONTENT_TYPE_APP_JSON)
+    mediaType = ExtendedSchemaNode(
+        String(), example=CONTENT_TYPE_APP_JSON)
```

Comment:
remove the extra newline for both `ExtendedSchemaNode` instantiation

Reply:
please also add a comment (**not** in docstring to avoid reporting this impl. detail in API, but code commentt) above each case regarding why it is overridden from corresponding `FormatMimeType` and `Format` (ie: because oneOf always fails because their `default=` value are always generating the fields, so both always present)